### DISCUSSION TITLE
Redo Button

### DISCUSSION THREAD

**Discussion by eitherlucas** [wkentaro/labelme#1135] [Ideas / Feature requests]
do you have plans on adding a redo button?

**Comment by armenforget** [comment]
I had labeled a very intricate polygon. Then when to edit mode and accidentally hit CTRL-Z. The entire polygon disappeared (not just the last point drawn) and I could not Redo to get it back. This feature would help avoid situations like that.

**Comment by rjbray915** [comment]
This would be an awesome QOL feature! The "undo" is right next to "delete" in the right-click menu, so I accidentally undo the last polygon I drew instead of deleting.
